icons/meson.build

```diff
@@ -0,0 +1,21 @@
+install_data([
+    'spnavcfg-48x48.png'
+  ],
+  install_dir: 'share/icons/hicolor/48x48'
+)
```

Comment:
These all need to use the `rename:` kwarg, and install to the `apps/` subdirectory of your current implementation.

Consider using a foreach loop instead. https://github.com/FreeSpacenav/spnavcfg/blob/97673e7f9d57b1d99ec4484b487ad4d797d2cd7d/Makefile.in#L36-L40

In meson, this is:

```meson
foreach size: ['48x48', '128x128', '256x256']
    install_data(
        'spnavcfg-@0@.png'.format(size),
        rename: 'spnavcfg.png',
        install_dir: get_option('datadir') / 'icons/hicolor' / size / 'apps')
endforeach

Reply:
yes, I use that loop in several projects, thanks!